packages/compute-baseline/src/baseline/index.ts

```diff
@@ -123,24 +130,12 @@ export function computeBaseline(
  * Compute the Baseline support ("high", "low" or false, dates, and releases)
  * for a single compat key.
  */
-function calculate(compatKey: string, compat: Compat): SupportDetails {
+function calculate(compatKey: string, compat: Compat) {
   const f = feature(compatKey);
-  const s = support(f, browsers(compat));
-  const keystoneDate = findKeystoneDate([...s.values()]);
-
-  const { baseline, baseline_low_date, baseline_high_date, discouraged } =
-    keystoneDateToStatus(keystoneDate, f.deprecated ?? false);
```

Comment:
Eliminating this call to `keystoneDateToStatus` made this change a bit simpler, and the information wasn't used anyway.